spacy/tests/README.md

```diff
@@ -10,12 +10,16 @@ Tests for spaCy modules and classes live in their own directories of the same na
 
 ## Table of contents
 
-1. [Running the tests](#running-the-tests)
-2. [Dos and don'ts](#dos-and-donts)
-3. [Parameters](#parameters)
-4. [Fixtures](#fixtures)
-5. [Helpers and utilities](#helpers-and-utilities)
-6. [Contributing to the tests](#contributing-to-the-tests)
+- [spaCy tests](#spacy-tests)
```

Comment:
What happened to the formatting here? Was this intentional?

Reply:
Oh, I didn't realize, `prettier` formatted the table of contents automatically to this. I've replaced the old table and saved without formatting.